input/pagecontent/shorthand.md

```diff
@@ -0,0 +1,924 @@
+
+> **NOTE:** All extensions have the same structure, but extensions can either have a value (i.e. a value[x] element) or sub-extensions, but not both. To create a complex extension, the extension array of the extension must be sliced (see example, below).
+
+**Example** Define simple extension:
+
+```
+Extension:      USCoreBirthSexExtension 
+Id:             us-core-birthsex
+Title:          "US Core Birth Sex Extension"
+Description:     "A code classifying the person's sex assigned at birth"
+// publisher, contact, and other metadata here using carat (^) syntax (omitted)
+* value[x] only code
+* valueCode from BirthSex (required)
+```
+
+**Example** Define complex extension:
+```
+Extension:      USCoreEthnicityExtension
+Id:             us-core-ethnicity
+Title:          "US Core Ethnicity Extension"
+Description:    "Concepts classifying the person into a named category of humans sharing common history, traits, geographical origin or nationality. "
+* extension contains 
+    ombCategory 0..1 MS and 
+    detailed 0..* and 
+    text 1..1 MS
+// inline definition of sub-extensions
+* extension[ombCategory] ^short = "Hispanic or Latino|Not Hispanic or Latino"
+* extension[ombCategory].value[x] only Coding
+* extension[ombCategory].valueCoding from OmbEthnicityCategories (required)
+* extension[detailed] ^short = "Extended ethnicity codes"
+* extension[detailed].value[x] only Coding
+* extension[detailed].valueCoding from DetailedEthnicity (required)
+* extension[text] ^short = "Ethnicity text"
+* extension[text].value[x] only string
+```
+
+**Example**  Extension with explicit parent:
+
+```
+Extension:      BinaryBirthSexExtension
+Parent:         USCoreBirthSexExtension
+Id:             binary-birthsex
+Title:          "Binary Birth Sex Extension"
+Description:     "As of 2019, certain US states only allow M or F on birth certificates."
+* valueCode from BinaryBirthSex (required)
+```
+
+### Defining Mappings
+
+[Mappings to other standards](https://www.hl7.org/fhir/mappings.html) are an optional part of a SD. These mappings are informative and are provided to help implementers understand the content of the SD and use the profile or resource correctly. While it is possible for profile authors to include mappings using escape syntax, FSH provides a more modular approach.
+
+> **NOTE:** The informational mappings in SDs should not be confused with functional mappings provided by [FHIR Mapping Language](https://www.hl7.org/fhir/mapping-language.html) and the [StructureMap resource](https://www.hl7.org/fhir/structuremap.html).
+
+To create a mapping, the keywords `Mapping`, `Source`, `Target` and `Id` are used. Any number of [mapping rules](#mapping-rules) then follow.
+
+**Example**  // MK -- NOTE -- I made some changes -- to be checked
```

Comment:
@cmoesel he made some changed -- to be checked

 (I don't know enough about mappings to check)